script/StructFilter.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Struct Filter
+ *
+ * remove toggling inherited by reusing core search form
+ */
+jQuery('fieldset.struct-filter-form.search-form button.toggleAssistant').remove();
+jQuery('fieldset.struct-filter-form.search-form .advancedOptions').show();
```

Comment:
These come from reusing the  default search styles? Can we avoid adding and removing stuff here?